test/catch.hpp

```diff
@@ -11310,7 +11310,7 @@ namespace Catch {
     std::string TagInfo::all() const {
         size_t size = 0;
         for (auto const& spelling : spellings) {
-            // Add 2 for the brackes
+            // Add 2 for the brackets
```

Comment:
Things like this are frustrating -- it could be `braces` and it isn't terribly obvious, but I'm moderately confident because later in the file it does compose `[` + ... + `]`.